src/ekore/operator_matrix_elements/polarized/space_like/as2.py

```diff
@@ -159,12 +159,11 @@ def A_hg(n, cache, L):
     S1 = c.get(c.S1, cache, n)
     S2 = c.get(c.S2, cache, n)
     S3 = c.get(c.S3, cache, n)
-    Sm21 = c.get(c.Sm21, cache, n, is_singlet=True)
+    Sm21 = c.get(c.Sm21, cache, n, is_singlet=False)
```

Comment:
I know you reverted here to the previous version, but isn't this strange? $A_{hg}$ is for sure a singlet-like quantity (there is the gluon in the name), but you are calling $S_{-2,1}$ on the non-singlet branch?

Reply:
But for polarised odd and even moments are swapped no? 
My understanding is that they have computed odd moments here.

Reply:
yes, I think you're right - note to myself: rename the parameter during rustification to `is_even` instead of `is_singlet`